NuGet.Config

```diff
@@ -7,7 +7,7 @@
     <add key="dotnet-libraries" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-libraries/nuget/v3/index.json" />
     <add key="myget-legacy@Local" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy%40Local/nuget/v3/index.json" />
     <add key="nuget-build" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/nuget-build/nuget/v3/index.json" />
-    <add key="vside" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/msft_consumption%40Local/nuget/v3/index.json" />
```

Comment:
I am wondering, in the worst-case scenario, if having the `%40Local` would be helpful because we don't run restore in interactive mode, the tooling can't respond to 401. If we experience this issue again in the future and we don't have `%40Local` in the feed name, then someone from the team has to run the restore locally so that the feed can save the package from the upstream to the local view. Once the local restore succeeds, the restore will also succeed on CI because the local view has the package injected from the upstream.

Reply:
I've un-reverted the change, so we will keep `@local` as it was.